packages/snjs/lib/utils.ts

```diff
@@ -540,3 +545,27 @@ export function isSameDay(dateA: Date, dateB: Date) {
     dateA.getDate() === dateB.getDate()
   );
 }
+
+ /**
+   * Sorts an array of objects in natural order
+   * @param items - The array of objects to sort
+   * @param property - The objects' property to sort by
+   * @param direction - The sorting direction, either ascending (default) or descending
+   * @returns Array of objects sorted in natural order
+   */
+export function naturalSort(items: any[], property: string, direction: 'asc' | 'desc' = 'asc'): any[] {
```

Comment:
Can use generics here? <T> and keyof T for property.

Reply:
Done! 👌